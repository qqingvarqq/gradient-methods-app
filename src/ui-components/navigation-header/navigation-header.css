.navigation-header {
  display: flex;
  height: 64px;
  align-items: center;
  justify-content: flex-start;
}
