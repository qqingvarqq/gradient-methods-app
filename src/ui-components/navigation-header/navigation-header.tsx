import * as React from 'react';
import styles from './navigation-header.module.css';

export const NavigationHeader: React.FC<React.DetailedHTMLProps<
  React.HTMLAttributes<HTMLDivElement>,
  HTMLDivElement
>> = (props) => {
  const {className = '', ...rest} = props;
  return <div className={`${styles.root} ${className}`} {...rest}></div>;
};
