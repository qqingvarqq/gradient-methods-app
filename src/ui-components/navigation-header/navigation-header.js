import React from 'react';
import './navigation-header.css';

export default function NavigationHeader(props) {
  const {className, ...rest} = props;
  const classNames = 'navigation-header ' + (className || '');
  return <div className={classNames} {...rest}></div>;
}
