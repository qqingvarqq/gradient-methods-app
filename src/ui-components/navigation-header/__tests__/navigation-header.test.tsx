import * as React from 'react';
import {render} from '@testing-library/react';
import {NavigationHeader} from '../navigation-header';

describe('NavigationHeader', () => {
  it('should render without crashing', () => {
    const {asFragment} = render(<NavigationHeader />);
    expect(asFragment()).toMatchSnapshot();
  });
});
