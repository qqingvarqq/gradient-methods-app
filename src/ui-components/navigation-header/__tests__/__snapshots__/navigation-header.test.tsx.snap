// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavigationHeader should render without crashing 1`] = `
<DocumentFragment>
  <div
    class="root "
  />
</DocumentFragment>
`;
